services/dashboard/services/models/models.js

```diff
@@ -45,29 +45,34 @@ angular.module('dashboard')
         args = args || {};
         return restapi.get(path).then(function(response) {
           var oldModel;
-          var model = decodeFn(response.data, args);
 
-          model.$subscribe = function(scope, onData, onError) {
-            restapi.subscribe(args.pathOverride || path, scope, function(data) {
-              try {
-                var newModel = decodeFn(data, args);
-                if (!_.isEqual(newModel, oldModel)) {
-                  oldModel = newModel;
-                  return onData(newModel);
-                }
-              } catch (ex) {
-                if (angular.isFunction(onError)) {
-                  return onError(data);
+          if (response != null) {
```

Comment:
Can we let it fail? As designed, the initial response cannot be invalid. In your code, it will return an `undefined`, but we do not see anything in console, this does not help anything.